satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -616,6 +616,31 @@ read all (
 	where storagenode_storage_tally.interval_end_time >= ?
 )
 
+//--- peer_identity ---//
+
+model peer_identity (
+    key node_id
+
+    field node_id        blob
+    field serial_number  blob (updatable)
```

Comment:
Can we add a comment here or somewhere else that makes it clear that the serial number is from the leaf?

Reply:
renamed the field to `leaf_serial_number`